charts/omnirpc/templates/deployment.yaml

```diff
@@ -53,9 +53,10 @@ spec:
             {{- toYaml .Values.env | nindent 12 }}
           {{- end }}
           ports:
-            - name: http
-              containerPort: 5080
-              protocol: TCP
+          {{- range .Values.service.ports }}
```

Comment:
do we want to export the metrics at the service level, or the container level? Not sure which we're doing here but container is definitely preferable

Reply:
service level is more standard pod level should be used only if we don't have a service

Reply:
I don't think so. The container label exists for a reason by default on prom. This way you can do aggegates across all your containers.

If you do service level you're just hitting a random container every scrape interval and you miss any outliers in your histogram